lib/mv_opentelemetry/dataloader.ex

```diff
@@ -27,23 +28,55 @@ defmodule MvOpentelemetry.Dataloader do
     :ok
   end
 
-  def handle_event([:dataloader, :source, :batch, :run, :start], _measurements, meta, opts) do
+  def handle_event(
+        [:dataloader, :source, :batch, :run, :stop],
+        measurements,
+        %{batch: batch},
+        opts
+      ) do
     event_name = "dataloader.source.batch.run"
 
-    parent_context = OpentelemetryProcessPropagator.fetch_parent_ctx(1, :"$callers")
-    attach_context(parent_context)
+    batch_attributes =
+      case batch do
+        {{:queryable, _pid, queryable, cardinality, col, _}, _} ->
```

Comment:
Some more attributes for DL, not sure how useful these are, but maybe we can do something with them.